src/js/timingDiagram.js

```diff
@@ -1,31 +1,32 @@
-// draws cromwell timing diagram for a submission  Cromwell 19
+// draws cromwell timing diagram for a submission. Last synced with Cromwell 34.
 
-function addDataTableRow(dataTable, callName, callPartName, startDate, endDate) {
+// ======================= START code used as-is from Cromwell =======================
+/*
+    following code, up until the END marker, is copied directly from Cromwell WITH THREE
+    EXCEPTIONS. Each of the three exceptions is preceded by a comment. These comments
+    all start with "DA:" and end with "for FireCloud."
+*/
+var parentWorkflowNames = [];
+var expandedParentWorkflows = [];
+var chartView;
+
+function addDataTableRow(dataTable, callName, callPartName, startDate, endDate, ancestry) {
```

Comment:
Maybe a quick comment somewhere about what ancestry is?

Reply:
added a comment at line 9.